cni/87-podman-bridge.conflist

```diff
@@ -2,28 +2,37 @@
     "cniVersion": "0.4.0",
     "name": "podman",
     "plugins": [
-      {
-        "type": "bridge",
-        "bridge": "cni0",
-        "isGateway": true,
-        "ipMasq": true,
-        "ipam": {
-            "type": "host-local",
-            "subnet": "10.88.0.0/16",
-            "routes": [
-                { "dst": "0.0.0.0/0" }
-            ]
-        }
-      },
-      {
-        "type": "portmap",
-        "capabilities": {
-          "portMappings": true
-        }
-      },
-      {
-        "type": "firewall",
-	"backend": "iptables"
-      }
+	{
+            "type": "bridge",
+            "bridge": "cni-podman0",
+            "isGateway": true,
+            "ipMasq": true,
+            "ipam": {
+		"type": "host-local",
+		"routes": [
+		    {
+			"dst": "0.0.0.0/0"
+		    }
+		],
+		"ranges": [
+		    [
+			{
+			    "subnet": "10.88.0.0/16",
+			    "gateway": "10.88.0.1"
```

Comment:
this technically is not required for our default config

Reply:
Eh, I figure it probably doesn't hurt. We can remove it if you want.